--- a/quickshell/dms/Modules/DankBar/Widgets/AppsDock.qml
+++ b/quickshell/dms/Modules/DankBar/Widgets/AppsDock.qml
@@ -417,7 +417,7 @@
     Component {
         id: rowLayout
         Row {
-            spacing: Theme.spacingXS
+            spacing: Theme.spacingS
 
             Repeater {
                 id: repeater
@@ -434,7 +434,7 @@
     Component {
         id: columnLayout
         Column {
-            spacing: Theme.spacingXS
+            spacing: Theme.spacingS
 
             Repeater {
                 model: ScriptModel {
